packages/storybook/stories/table.stories.tsx

```diff
@@ -496,18 +498,35 @@ export const SortableRows: Story = () => {
             accessorKey: 'column3',
             sortable: true,
         },
+        {
+            header: 'Custom sort by length',
+            accessorKey: 'column4',
+            sortable: true,
+            sortingFn: (rowA, rowB, columnId: string) => {
+                const a = rowA.original[columnId] as string;
+                const b = rowB.original[columnId] as string;
```

Comment:
Le probleme c'est qu'il accede l'object avec un key "string", donc possiblement n'importe quoi.

Y a une couple de facon de typer ca, mais le plus simple c'est juste d'ecrire directement le key.

```suggestion
            sortingFn: (rowA, rowB) => {
                const a = rowA.original.column4;
                const b = rowB.original.column4;
```